Statut_HSP.tex

```diff
@@ -168,19 +174,47 @@ \section{Członkowie Stowarzyszenia}
   regularnego opłacania składek.
 \end{enumerate}
 
-\myparagraph{§ 14}
+\myparagraph{§ 12.NZ.1}
+\begin{enumerate}
+\item
+  Prawa i obowiązki każdego członka nadzwyczajnego z osobna definiuje zarząd.
+\item
+  Prawa i obowiązki członka nadzwyczajnego mogą być zmienione na jego wniosek po zatwierdzeniu przez zarząd.
+\item
+  Prawa członka nadzwyczajnego muszą zawierać się w zbiorze praw członka zwyczajnego.
```

Comment:
To definiuje, że nie można nadać dodatkowych uprawnień / obowiązków członkowi nadzwyczajnemu. To mogło by być potencjalnie przydatne, ale tak jak jest też jest ok.

Reply:
zostawie na razie - bo to, ze zarząd mógłby dać uprawnienia wyższe niż ma byłoby przerażające